src/pkg/util/fs/helper_linux_test.go

```diff
@@ -45,3 +47,23 @@ func TestIsSuid(t *testing.T) {
 		t.Errorf("IsSuid returns false for /bin/su setuid bit")
 	}
 }
+
+func TestRootDir(t *testing.T) {
+	var tests = []struct {
+		path     string
+		expected string
+	}{
+		{"/path/to/something", "/path"},
+		{"relative/path", "relative"},
+		{"/path/to/something/", "/path"},
+		{"relative/path/", "relative"},
+		{"/path", "/path"},
+		{"/path/", "/path"},
```

Comment:
Is there a case to have tests here which include . and .. components?

Reply:
Absolutely. So the reason this file is commented out currently is because I'm still pondering the best way to handle the testing, as well as what the interface to the package should be. Once I figure those things out, I'll definitely add some tests which include `.` and `..`. Thanks for the suggestion